ui/src/components/RoleScopesTable/index.jsx

```diff
@@ -93,18 +93,20 @@ export default class RoleScopesTable extends Component {
 
     return (
       <Fragment>
-        <ListItem
-          style={style}
+        <TableCellItem
           button
           component={Link}
           to={
             selectedScope
               ? `/auth/roles/${encodeURIComponent(item)}`
               : `/auth/scopes/${encodeURIComponent(item)}`
           }>
-          <ListItemText primary={item} />
+          <ListItemText
+            disableTypography
+            primary={<Typography>{item}</Typography>}
+          />
           <LinkIcon size={iconSize} />
-        </ListItem>
+        </TableCellItem>
```

Comment:
![Screen Shot 2019-10-07 at 3 14 12 PM](https://user-images.githubusercontent.com/3766511/66341385-26707500-e915-11e9-945c-3b97810c897b.png)

Observations:
* A horizontal line is showing up in the middle which doesn't happen in the clients tab
* The text is too close to the border

Reply:
Thank you for the feedback @helfi92 I will make those changes right away!